src/contexts/navigation/GalleryNavigationProvider.tsx

```diff
@@ -21,6 +22,12 @@ type Props = {
 
 export function GalleryNavigationProvider({ children }: Props) {
   const [historyStackLength, setHistoryStackLength] = useState(0);
```

Comment:
now that we have `historyStack`, i think we can toss `historyStackLength`, since it's essentially a redundant piece of state.

down below we use `historyStackLength` to determine `canGoBack`:
![image](https://user-images.githubusercontent.com/12162433/152407420-677a1167-e180-4899-9822-3e7a27750496.png)

but we can just update this to:
```
export function useCanGoBack() {
  const { historyStack } = useGalleryNavigationContext();

  return historyStack.length > 0;
}
```